_pages/learning.md

```diff
@@ -33,3 +33,16 @@ Get your hands on Crystal with a series of exercises, and compare your solution
 to that of the many members who tried already.
 
 [Crystal track on Exercism](https://exercism.org/tracks/crystal/)
+
+<span id="codecrafters"></span>
+
+### CodeCrafters
+
+Keep your Crystal skills sharp by learning how to build real things. Practice interesting programming challenges like building your own Git, Redis, Docker, and even SQLite from scratch.
+
+> **NOTE:**
+> Registration to CodeCrafters is free. With the link below you can get a 40% discount when upgrading to an annual paid plan.
+>
+> If you opt to pay, 60% of the income will come back to the Crystal team to help developing Crystal!
+
+[Discounted access](https://app.codecrafters.io/join?via=crystal-team)
```

Comment:
**suggestion:** Put the callout below the link.
The callout block puts quite a distance between the description and the call to action link. It also leads to a different shape than the other entries on this site. This all means the link is not very obvious and might be missed.

![grafik](https://github.com/crystal-lang/crystal-website/assets/466378/b49fec21-f293-4587-84d5-ec5aa709085c)